units/INU2003/INU2003_unit.bp

```diff
@@ -62,7 +61,6 @@ UnitBlueprint {
         'LAND',
         'TECH2',
         'INDIRECTFIRE',
-        'SILO',

```

Comment:
i remove it by mistake :/ and now its too much work to make this stupidly huge comit again. After merge make new comit that get it back.